examples/mixed_types.brainrot

```
@@ -1,5 +1,6 @@
 skibidi main {
     chad f = 3.14;
+    bruh;
```

shouldn't this throw an error, since `break` cannot be used outside a loop in C?

---

yeah, you right i'll fix it

---

it actually  through an error but some how the test didn't get it

---

Yes, it is because of the const test that should throw an error. I'll think of a better way of handling tests that should output stderr.

---

For example, the `baka` test now is disabled because it outputs to stderr.